### PR TITLE
updated quota monitoring CF doc

### PR DESCRIPTION
**Pull request by maunope**
Update images, updated metrics format, removed "any metric" from alert description

### CODE REVIEW

File: blueprints/cloud-operations/quota-monitoring/README.md [hunk 1]
```diff
@@ -10,11 +10,25 @@ Regardless of its specific purpose, this blueprint is also useful in showing how
 
 The solution is designed so that the Cloud Function arguments that control function execution (eg to set which project quotas to monitor) are defined in the Cloud Scheduler payload set in the PubSub message, so that a single function can be used for different configurations by creating more schedules.
 
-Quota time series are stored using a [custom metric](https://cloud.google.com/monitoring/custom-metrics) with the `custom.googleapis.com/quota/gce` type and [gauge kind](https://cloud.google.com/monitoring/api/v3/kinds-and-types#metric-kinds), tracking the ratio between quota and limit as double to aid in visualization and alerting. Labels are set with the quota name, project id (which may differ from the monitoring workspace projects), value, and limit. This is how they look like in the metrics explorer.
+Quota time series are stored using  [custom metrics](https://cloud.google.com/monitoring/custom-metrics) with metric type for usage, limit and utilization; metric types are named using a common frefix and two tokens joined by a `-` character:
```
Comment (ludoo) [inline blueprints/cloud-operations/quota-monitoring/README.md:13]: `prefix`

**Review by ludoo — APPROVED** [review]

File: blueprints/cloud-operations/quota-monitoring/README.md (hunk 1 above)
Comment (maunope) [reply blueprints/cloud-operations/quota-monitoring/README.md]: pushed fix

File: blueprints/cloud-operations/quota-monitoring/main.tf [hunk 2]
```diff
@@ -111,9 +111,9 @@ resource "google_monitoring_alert_policy" "alert_policy" {
   display_name = "Quota monitor"
   combiner     = "OR"
   conditions {
-    display_name = "simple quota threshold"
+    display_name = "simple quota threshold for cpus utilization"
     condition_threshold {
-      filter          = "metric.type=\"custom.googleapis.com/quota/gce\" resource.type=\"global\""
+      filter          = "metric.type=\"custom.googleapis.com/quota/cpus_utilization\" resource.type=\"global\""
```
Comment (ludoo) [inline blueprints/cloud-operations/quota-monitoring/main.tf:116]: I would use singular: `cpu_utilization`

**Review by ludoo — APPROVED** [review]

File: blueprints/cloud-operations/quota-monitoring/main.tf (hunk 2 above)
Comment (maunope) [reply blueprints/cloud-operations/quota-monitoring/main.tf:116]: it's a metric name, we can't change it

Comment (ludoo) [reply blueprints/cloud-operations/quota-monitoring/main.tf:116]: yow, awful choice :)

**Pull request merged**